Kitodo/src/main/webapp/WEB-INF/templates/includes/system/confirmWorkflowPopup.xhtml

```diff
@@ -21,17 +21,16 @@
                   width="400px"
                   modal="true"
                   dynamic="true">
-            <h:form id="confirmWokflowForm">
-                <p:panelGrid id="passwordGrid" columns="1" layout="grid">
+            <h:form id="confirmWorkflowForm">
+                <p:panelGrid columns="1" layout="grid">
                     <p:row>
 
                         <div>
                             <p:commandButton id="useExistingWorkflow"
-                                             value="#{msgs.cancel}"
```

Comment:
Don't we still need a `cancel` button in the dialog if we decide to neither use an existing workflow nor create a new one but really just want to cancel the migration process and close the dialog? We don't have "close"-buttons on our dialogs after all.

Reply:
added